src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCSVFileRecord.java

```diff
@@ -49,6 +49,10 @@
      */
     private final String delimiter;
 
+    private boolean escapeDelimiters;
+
+    private static final String escapeSplitPattern = "(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)";
```

Comment:
Can we add a comment here that explains this regex briefly?

Reply:
Sure.